include/swift/AST/Module.h

```diff
@@ -744,7 +748,7 @@ class FileUnit : public DeclContext {
   // Only allow allocation of FileUnits using the allocator in ASTContext
   // or by doing a placement new.
   void *operator new(size_t Bytes, ASTContext &C,
-                     unsigned Alignment = alignof(FileUnit));
+                     unsigned Alignment = alignof(FileUnit&));
```

Comment:
Maybe we can do something silly like declaring a function `alignOfFileUnit` and then defining it at the bottom of the file as static inline.